casr/src/bin/casr-libfuzzer.rs

```diff
@@ -81,6 +78,18 @@ fn main() -> Result<()> {
                 .long("no-cluster")
                 .help("Do not cluster CASR reports")
         )
+        .arg(Arg::new("san-force")
+            .long("san-force")
+            .action(ArgAction::SetTrue)
+            .help("Force casr-san run without sanitizers symbols check"))
+        .arg(
+            Arg::new("casr-gdb-args")
+                .long("gdb")
+                .action(ArgAction::Set)
+                .num_args(1..)
+                .value_terminator(";")
+                .help("Specify casr-gdb target arguments to add casr reports for uninstrumented binary"),
```

Comment:
can we just accept a large string separated with spaces?

Reply:
like

```
--casr-gdb-args '/fuzz_target -g "second arg"'
```

Reply:
Also, consider escaping of `\"`

Reply:
done